app/domain/people/neuanmeldungen/base.rb

```diff
@@ -33,6 +37,10 @@ def paying_person?(role)
 
     private
 
+    def family_neuanmeldungs_role?(person)
+      person.roles.where(type: NEUANMELDUNGEN_ROLES.map(&:sti_name), group: group).family.any?
+    end
```

Comment:
I'm a bit confused, that we need to search through the roles again (I assumed the primary role on the person that we work on should be available anyways) but it doesn't seem to be the case.